lib/slop/options.rb

```diff
@@ -77,6 +78,18 @@ def each(&block)
       options.each(&block)
     end
 
+    # The block is called when there is an error when processing the options.
+    #
+    # You can use it to print the help or the specific error and terminate the program execution.
+    # Example:
```

Comment:
Here's a documentation formatting suggestion:

`Example:` => `@example Print the help for the specific error, and then terminate the program.`

This change makes it a YARD annotation and will show up as a Ruby-syntax-highlighted example snippet, with that heading.

Reply:
Actually I'm not convinced about this. Slop uses something closer to [TomDoc](http://tomdoc.org) than YARD. I'd be happy to switch to YARD (I understand its superiority though am not a fan of the syntax) if that's the consensus, but we should change the docs everywhere if we decide to do that

Reply:
I'm leaving it as it is. OK?